README.md

```diff
@@ -1 +1,4 @@
-# ia-home-office-integration-api
+# Home Office Integration Service API
```

Comment:
Let's just call it the Home Office Integration API

Reply:
Corrected